components/support/ktx/src/main/java/mozilla/components/support/ktx/android/util/DisplayMetrics.kt

```diff
@@ -17,3 +17,11 @@ fun Int.dpToFloat(displayMetrics: DisplayMetrics) = TypedValue.applyDimension(
  * Converts a value in density independent pixels (dp) to the actual pixel values for the display.
  */
 fun Int.dpToPx(displayMetrics: DisplayMetrics) = dpToFloat(displayMetrics).toInt()
+
+/** Converts a value in density independent pixels (dp) to a px value. */
+fun Float.dpToPx(displayMetrics: DisplayMetrics) = TypedValue.applyDimension(
+    TypedValue.COMPLEX_UNIT_DIP, this, displayMetrics)
+
+/** Converts a value in scale independent pixels (sp) to a px value. */
+fun Float.spToPx(displayMetrics: DisplayMetrics) = TypedValue.applyDimension(
+    TypedValue.COMPLEX_UNIT_SP, this, displayMetrics)
```

Comment:
Should we add some tests for this new utilities? it will be nice to mentioning them in the changelog too :)

Reply:
Not really sure how we could test this. 🤔

Reply:
You could assert that value you are getting from `spToPx` will be the same as executing 
```TypedValue.applyDimension(TypedValue.COMPLEX_UNIT_SP, this, displayMetrics)```,  as `spToPx` is a shortcut

Reply:
I thought of that, but it would look like we are just testing how extensions work. Or maybe I am seeing it wrong.

Reply:
The idea is we want to assert the algorithm of converting a `Float` from `sp` to `px` will return the same value as using `spToPx`, if there are other ways to convert from an `sp` to `px` we can use it too.

Reply:
Good point! Thanks!

Reply:
@Amejia481 Added the tests, let me know what you think.